articles/active-directory/authentication/howto-mfa-nps-extension-rdg.md

```diff
@@ -223,7 +223,7 @@ To ensure there is time to validate users’ credentials, perform two-step verif
 By default, when you configure the RD Gateway to use a central policy store for connection authorization policies, the RD Gateway is configured to forward CAP requests to the NPS server. The NPS server with the Azure MFA extension installed, processes the RADIUS access request. The following steps show you how to verify the default connection request policy. 
 
 1. On the RD Gateway, in the NPS (Local) console, expand **Policies**, and select **Connection Request Policies**.
-2. Right-click **Connect Request Policies**, and double-click **TS GATEWAY AUTHORIZATION POLICY**.
+2. Double-click **TS GATEWAY AUTHORIZATION POLICY**.
```

Comment:
No need to right-click. By selecting "Connection Request Policies" on the left-hand side, "TS GATEWAY AUTHORIZATION POLICY" is available on the right-hand side.